### PR TITLE
Bump deps in aleph-client.

### PR DESCRIPTION
**Pull request by deuszx**

### CODE REVIEW

**Review by obrok — APPROVED** [review]
If tests pass, then cowabunga!

**Review by mike1729 — APPROVED** [review]

File: aleph-client/Cargo.toml [hunk 1]
```diff
@@ -10,11 +10,10 @@ anyhow = "1.0"
 codec = { package = 'parity-scale-codec', version = "3.0.0", features = ['derive'] }
 hex = { version = "0.4.3", features = ["alloc"] }
 log = "0.4"
+thiserror = "1.0.39"
```
Comment (fbielejec) [inline aleph-client/Cargo.toml:13]: this change makes little sense - specifying 1.0 will already pull that version if it's the newest

Comment (deuszx) [reply aleph-client/Cargo.toml:13]: You're right. Is it okay if I leave it though or would you prefer for me to revert it?

Comment (fbielejec) [reply aleph-client/Cargo.toml:13]: revert please :raised_hands:

Comment (deuszx) [reply aleph-client/Cargo.toml]: Reverted.

**Pull request merged**